torch/overrides.py

```diff
@@ -533,7 +533,6 @@ def get_testing_overrides() -> Dict[Callable, Callable]:
         torch.logical_not: lambda input, out=None: -1,
         torch.logical_or: lambda input, other, out=None: -1,
         torch.logical_xor: lambda input, other, out=None: -1,
-        torch.logsumexp: lambda input, names, keepdim=False, out=None: -1,
```

Comment:
You dont need to remove this one.

Reply:
Is there a reason why there are 2 same entries? I have added it back, but just asking to clarify. :)

Reply:
We are not actually removing `torch.logsumexp`. 

Also this test verifies if all the exposed function in the `torch` and few other namespaces have an entry for this test (so it would have failed mostly likely with this removed).